### DISCUSSION TITLE
[Need HELP] Errors after import: Conflicting lifecycle mapping metadata

### DISCUSSION THREAD

**Discussion by ricemaster** [dbeaver/dbeaver#20578] [DBeaver Development]
<img width="1211" alt="iShot_2023-07-15_17 46 33" src="https://github.com/dbeaver/dbeaver/assets/10449002/ebfcb114-7a9f-4f7d-9ed0-c22e61282790">
100+ errors after import project

`Conflicting lifecycle mapping metadata (project packaging type="eclipse-feature"): Mapping defined in 'org.eclipse.m2e.pde.connector_2.1.300.20230520-0934 [304]' and 'org.sonatype.tycho.m2e_0.10.0.20220926-1324 [671]'. To enable full functionality, remove the conflicting mapping and run Maven->Update Project Configuration.`

Any help would be appreciated, thanks~~

**Comment by LonwoLonwo** [comment]
As a workaround.

You need to create in the eclipse-workspace/.metadata/.plugins/org.eclipse.m2e.core the file lifecycle-mapping-metadata.xml with following data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<lifecycleMappingMetadata>
  <lifecycleMappingFilters>
    <lifecycleMappingFilter>
      <symbolicName>org.eclipse.m2e.pde.connector</symbolicName>
      <versionRange>[2.1.2,)</versionRange>
      <packagingTypes>
        <packagingType>eclipse-test-plugin</packagingType>
        <packagingType>eclipse-plugin</packagingType>
        <packagingType>eclipse-feature</packagingType>
      </packagingTypes>
    </lifecycleMappingFilter>
  </lifecycleMappingFilters>
</lifecycleMappingMetadata>
```

**Comment by StefanG12345** [comment]
you just saved my day, thanks
